packages/jellyfish-api-core/__tests__/category/loan/createLoanScheme.test.ts

```diff
@@ -0,0 +1,106 @@
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import { UTXO } from '@defichain/jellyfish-api-core/category/loan'
+import BigNumber from 'bignumber.js'
+import { Testing } from '@defichain/jellyfish-testing'
+
+describe('Loan', () => {
+  const container = new LoanMasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+
+    // NOTE(jingyi2811): Default scheme
+    await testing.rpc.loan.createLoanScheme(100, new BigNumber(1.5), { id: 'default' })
+    await testing.generate(1)
+  })
+
+  afterEach(async () => {
+    const data = await testing.container.call('listloanschemes')
+    const result = data.filter((d: { default: boolean }) => !d.default)
+
+    for (let i = 0; i < result.length; i += 1) {
+      // NOTE(jingyi2811): Delete all schemes except default scheme
+      await testing.container.call('destroyloanscheme', [result[i].id])
+      await testing.generate(1)
+    }
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should createLoanScheme', async () => {
```

Comment:
test create a duplicate scheme and
create a new scheme but same ratio with existing scheme as well

Reply:
See line 58 and 63